### PR TITLE
Cherry picking new notification

### PR DESCRIPTION
**Pull request by tidy-dev**
Part of #1685

## Description

A onetime popup to introduce users to cherry picking.

### Screenshots
Updated with image since video.
image

https://user-images.githubusercontent.com/75402236/110326471-b3c26800-7fe6-11eb-8131-31fa98c2bd39.mp4

## Release notes
Notes: no-notes

### CODE REVIEW

File: app/src/lib/stores/app-store.ts [hunk 1]
```diff
@@ -1731,6 +1739,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
       }
     })
 
+    this.hasShownCherryPickIntro = getBoolean(hasShownCherryPickIntroKey, true)
```
Comment (sergiou87) [inline app/src/lib/stores/app-store.ts]: Shouldn't the default be `false`? 🤔 
```suggestion
    this.hasShownCherryPickIntro = getBoolean(hasShownCherryPickIntroKey, false)
```

File: app/src/ui/history/commit-list.tsx [hunk 2]
```diff
@@ -182,6 +190,30 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
     return this.props.commitSHAs.findIndex(s => s === sha)
   }
 
+  private renderCherryPickIntroPopover() {
+    if (this.props.hasShownCherryPickIntro) {
+      return null
+    }
+
+    return (
+      <Popover caretPosition={PopoverCaretPosition.LeftTop}>
```
Comment (sergiou87) [inline app/src/ui/history/commit-list.tsx:206]: 😍  it's useful! 🎉

File: app/src/lib/stores/app-store.ts (hunk 1 above)
Comment (tidy-dev) [reply app/src/lib/stores/app-store.ts]: Hmm.. Yes.. but why did this work? 😄

Comment (tidy-dev) [reply app/src/lib/stores/app-store.ts]: 🤦  It doesn't.. I just already had the key...

Comment (sergiou87) [reply app/src/lib/stores/app-store.ts]: :magic: 🤣

**Review by sergiou87 — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**